npm-global-without-sudo.md

```diff
@@ -21,13 +21,28 @@ npm config set prefix "${HOME}/.npm-packages"
 Add the following to your `.bashrc`/`.zshrc`:
 
 ```sh
-NPM_PACKAGES="${HOME}/.npm-packages"
+NPM_PACKAGES="${XDG_DATA_HOME}/npm-packages"
 
-export PATH="$PATH:$NPM_PACKAGES/bin"
+if ! [ -d "$NPM_PACKAGES" ] && command -v npm >/dev/null 2>&1; then
```

Comment:
I don't think this `command -v npm` is necessary. Only people using npm would add all this anyway.

Reply:
Okay, the recent commit addresses it. The idea was to put this into your dotfiles and carry them around; some day, one might end up on a system with npm.